app/models/purl_fetcher/record_deletes.rb

```diff
@@ -7,9 +7,9 @@ def initialize(druid:, latest_change: nil)
     end
 
     def enqueue
-      Rails.configuration.target_urls_hash.keys.each do |target|
+      Settings.SERVICE_INDEXERS.each do |target|
         Delayed::Job.enqueue(
-          DiscoveryDispatcher::IndexingJob.new('delete', druid, target)
+          DiscoveryDispatcher::IndexingJob.new('delete', druid, target[0].to_s.downcase)
         )
```

Comment:
probably should add a comment here with an example of a "`target`" value. is it an Array? if so, shouldn't you be creating n jobs per target rather than the first one only?